pages/Manuscripts.xml

```diff
@@ -69,6 +69,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <p>It can be that you find out the same manuscript is present under 2 identifiers.</p>
             <p>You can fix this by merging the two records in one and deleting the one with less information. Remember when doing this to check for references to the id which will go missing, and update them to point to the other identifier.</p>
             </div>
+         <div type="level2">
+            <head>Differenciating between various levels of description</head>
+            <p>Following  <ref target="https://github.com/BetaMasaheft/Documentation/issues/2700"
```

Comment:
Including references to issues is rarely done in the guidelines so far. On the one hand, this is very helpful and improves transparency, on the other, it makes the text somewhat more complex and longer. Maybe a compromise could be to move the reference into paranthesis at the end of the paragraph ("(see issue XX)"), so that it can be skipped easier when skimming the guidelines.

Reply:
Link to the issue must be included in the description of the pull request, then it can always be found (not as title but as a link, or several links - you can edit the very first message in this pull request)

For better documentation it is always better to have each issue clearly linked to each change in github history, so it is not ideal to have several files for several issues in one pull request (we are not going to change it now, but for the future).

Please remove the links to the issues from the guidelines, this is not something we want.

Reply:
@eu-genia @thea-m I have followed the example for issue 368 (https://github.com/BetaMasaheft/Documentation/issues/368)
in https://betamasaheft.eu/Guidelines/?id=manuscripts
Is it a different case or should I change that part too?